mono/dis/main.c

```diff
@@ -626,16 +626,14 @@ dis_locals (MonoImage *m, MonoMethodHeader *mh, const char *ptr)
 		unsigned char flags = *(const unsigned char *) ptr;
 		unsigned char format = flags & METHOD_HEADER_FORMAT_MASK;
 		guint16 fat_flags;
-		guint32 local_var_sig_tok, max_stack, code_size, init_locals;
-		int hsize;
+		guint32 local_var_sig_tok, init_locals;
 
 		g_assert (format == METHOD_HEADER_FAT_FORMAT);
 		fat_flags = read16 (ptr);
 		ptr += 2;
-		hsize = (fat_flags >> 12) & 0xf;
-		max_stack = read16 (ptr);
+		/* max_stack = read16 (ptr); */
```

Comment:
No, it doesn't have side effects.

https://github.com/mono/mono/blob/master/mono/metadata/mono-endian.h